include/login.php

```diff
@@ -0,0 +1,43 @@
+<?php
+function attemptLogin($username, $password){
+    $result=dbQuery('
+        SELECT *
+        FROM users
+        WHERE username = :username
+        AND password = :password',
+        array ('username'=>$username,
+               'password'=>$password))->fetch();
+        return $result;
+}
+
+function getUser($userId){
+    $result = dbQuery('
+        SELECT *
+        FROM users
+        WHERE userId = :userId
+    ', array('userId' => $userId))->fetch();
+    return $result;
+}
+
+function checkIsAdmin($userId){
+    if(isLoggedIn()){
+        $result = dbQuery('
+            SELECT *
+            FROM users
+            WHERE admin = 1
+        ')->fetch();
+    return true;   
```

Comment:
As described near the top, I'd also make a function that redirects to the login page if this returns false so you don't need to wrap every page in a big if statement. The same goes for `isLoggedIn` below.

I think Eva has done this if you want to look at an example.